scripts/split_storage_migration.sh

```diff
@@ -118,6 +166,7 @@ function finish_split_storage_migration {
 }
```

Comment:
As we are working on the script, please add echo for the last lines of this function (161 and 164)